packages/webpack/get-config.js

```diff
@@ -35,13 +34,9 @@ module.exports = async function getConfig (limitConfig, check, output) {
       path: output
     },
     optimization: {
-      concatenateModules: !check.disableModuleConcatenation
-    },
-    resolve: {
-      plugins: [PnpWebpackPlugin]
-    },
-    resolveLoader: {
-      plugins: [PnpWebpackPlugin.moduleLoader(module)]
+      concatenateModules: !check.disableModuleConcatenation,
+      minimize: true,
```

Comment:
I'm not sure if this should be forced to `true` like this; it depends if we only want it to automatically happen when the `mode` is `production` and not `development`.

Reply:
You can remove it and see the tests changes